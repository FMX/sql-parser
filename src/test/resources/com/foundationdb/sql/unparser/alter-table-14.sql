ALTER TABLE test.t ALTER COLUMN c SET GENERATED ALWAYS AS IDENTITY