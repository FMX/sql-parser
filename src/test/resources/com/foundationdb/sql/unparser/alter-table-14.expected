ALTER TABLE test.t ALTER COLUMN c SET GENERATED ALWAYS AS IDENTITY (START WITH 1, INCREMENT BY 1)